docs/api-docs/channels/channels-storefront-tutorial.md

```diff
@@ -257,7 +199,7 @@ Accept: application/json
 
 ## Notification settings
 
-Include the `Notifications` [protected section](#protected-ui-sections) in the channel's `config_meta` object to render the BigCommerce provided **Notifications** tab on the channel's settings page.
+Include the `notifications` [protected section](#protected-ui-sections) in the Channel Menus' `bigcommerce_protected_app_sections` array to render the BigCommerce-provided **Notifications** tab on the channel's settings page.

```

Comment:
same as above: get rid of tab and provide more clarity on this. Recommended: "Include the `notifications` [protected section](#protected-ui-sections) in the Channel Menus' `bigcommerce_protected_app_sections` array to render the BigCommerce-provided **Notifications** menu item in the channel's navigation and corresponding settings page.